examples/mfem/bp1.hpp

```diff
@@ -115,7 +115,8 @@ class CeedMassOperator : public mfem::Operator {
     const int ir_order = 2*(order + 2) - 1; // <-----
     const mfem::IntegrationRule &ir =
       mfem::IntRules.Get(mfem::Geometry::SEGMENT, ir_order);
-    CeedInt nqpts, nelem = mesh->GetNE(), dim = mesh->SpaceDimension();
+    CeedInt nqpts, nelem = mesh->GetNE(), dim = mesh->SpaceDimension(),
```

Comment:
For BP1/BP3 we should have ncomp = 1, ncomp_geo = 3. I like your use of ncompx below.